packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -2621,6 +2632,11 @@ addEventListener('fetch', event => {});`
 			);
 			writeAssets(assets);
 			mockUploadWorkerRequest({
+				expectedEntry(entry) {
+					// Ensure that we have not included the watch stub in production code.
+					// This is only needed in `wrangler dev`.
+					expect(entry).not.toMatch(/modules-watch-stub\.js/);
```

Comment:
is it worth checking the watch stub *did* get added for `wrangler dev`?

Reply:
There is a test in the fixture that would fail if the stub was not there because the watchers would not be wired up: https://github.com/cloudflare/workers-sdk/blob/7ad8ddd732c31441c9d245d7edd36199b768fc54/fixtures/additional-modules/test/index.test.ts#L97